src/lxc/confile.c

```diff
@@ -229,29 +229,6 @@ static struct lxc_config_t config_jump_table[] = {
-	{ "lxc.net.ipv6.gateway",           true,  set_config_net_ipv6_gateway,           get_config_net_ipv6_gateway,           clr_config_net_ipv6_gateway,           },
-	{ "lxc.net.link",                   true,  set_config_net_link,                   get_config_net_link,                   clr_config_net_link,                   },
-	{ "lxc.net.l2proxy",                true,  set_config_net_l2proxy,                get_config_net_l2proxy,                clr_config_net_l2proxy,                },
-	{ "lxc.net.macvlan.mode",           true,  set_config_net_macvlan_mode,           get_config_net_macvlan_mode,           clr_config_net_macvlan_mode,           },
-	{ "lxc.net.ipvlan.mode",            true,  set_config_net_ipvlan_mode,            get_config_net_ipvlan_mode,            clr_config_net_ipvlan_mode,            },
-	{ "lxc.net.ipvlan.isolation",       true,  set_config_net_ipvlan_isolation,       get_config_net_ipvlan_isolation,       clr_config_net_ipvlan_isolation,       },
-	{ "lxc.net.mtu",                    true,  set_config_net_mtu,                    get_config_net_mtu,                    clr_config_net_mtu,                    },
-	{ "lxc.net.name",                   true,  set_config_net_name,                   get_config_net_name,                   clr_config_net_name,                   },
-	{ "lxc.net.script.down",            true,  set_config_net_script_down,            get_config_net_script_down,            clr_config_net_script_down,            },
-	{ "lxc.net.script.up",              true,  set_config_net_script_up,              get_config_net_script_up,              clr_config_net_script_up,              },
-	{ "lxc.net.type",                   true,  set_config_net_type,                   get_config_net_type,                   clr_config_net_type,                   },
-	{ "lxc.net.vlan.id",                true,  set_config_net_vlan_id,                get_config_net_vlan_id,                clr_config_net_vlan_id,                },
-	{ "lxc.net.veth.mode",              true,  set_config_net_veth_mode,              get_config_net_veth_mode,              clr_config_net_veth_mode,              },
-	{ "lxc.net.veth.pair",              true,  set_config_net_veth_pair,              get_config_net_veth_pair,              clr_config_net_veth_pair,              },
-	{ "lxc.net.veth.ipv4.route",        true,  set_config_net_veth_ipv4_route,        get_config_net_veth_ipv4_route,        clr_config_net_veth_ipv4_route,        },
-	{ "lxc.net.veth.ipv6.route",        true,  set_config_net_veth_ipv6_route,        get_config_net_veth_ipv6_route,        clr_config_net_veth_ipv6_route,        },
-	{ "lxc.net.veth.vlan.id",           true,  set_config_net_veth_vlan_id,           get_config_net_veth_vlan_id,           clr_config_net_veth_vlan_id,           },
-	{ "lxc.net.veth.vlan.tagged.id",    true,  set_config_net_veth_vlan_tagged_id,    get_config_net_veth_vlan_tagged_id,    clr_config_net_veth_vlan_tagged_id,    },
```

Comment:
`src/tests/oss-fuzz.sh` uses `config_jump_table` to put all the keys into the seed corpus (to make it easier for the fuzz target to cover the parser). With this part of the table gone it seems it should start using `config_jump_table_net` as well. It shouldn't block this PR though. I'll open a follow-up PR once this is merged.

Reply:
Yeah. (I once experimented with generating a perfect hash table (Probably still have a branch somewhere.) instead of using a jump table. That actually shouldn't be too difficult to implement by itself. I think the main thing was to make sure that devs have an easy way to regenerate the table via a script or sm.)